community/command-line/src/main/java/org/neo4j/commandline/admin/AdminCommand.java

```diff
@@ -71,5 +73,30 @@ public String name()
         public abstract AdminCommand create( Path homeDir, Path configDir, OutsideWorld outsideWorld );
     }
 
+    abstract class Blocker extends Service
```

Comment:
No reason for this to be an abstract class. The `AdminCommand.Provider` is usefully abstract, but this needn't be.